src/main/kotlin/no/nav/helse/journalforing/gateway/JournalPostRequest.kt

```diff
@@ -9,7 +9,15 @@ internal data class JournalPostRequest(
     val tema: String, /// OMS
     val datoMottatt: String, // yyyy-MM-dd'T'HH:mm:ssZ
     val kanal: String, // NAV_NO
-    val journalfoerendeEnhet: String
+    val journalfoerendeEnhet: String) {
+    val tilleggsopplysninger = listOf(
+        Tilleggsopplysning("k9.kilde", "DIGITAL_SØKNADSDIALOG")
```

Comment:
Hvordan blir dette for de dialogene som ikke er en søknad?
F.eks. `Overføring av omsorgsdager`  eller `Melding om ettersendelse `?

Reply:
hmm ja du kan så si, trenger ikke den oppløsningen akkurat nå men kanskje ryddig å differensiere på det uansett.
Må legge til en k9.type for å vite om det er ettersendelse og, så kan jo flette inn det og 👍🏿 

Og til spørsmålet over, ja, det skal det gjøre 💾

Reply:
Ok, flott :) 
For brukerdialog så spiller det vel ikke noe stor rolle (tror jeg) om man trenger å differensiere på det pr. nå.